.circleci/config.yml

```
@@ -253,6 +253,87 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: Scheduled tests failed on iOS device!'
+  build-android-js-bundle:
+    # This job builds the JS bundle that the Android react-navite-bridge
+    # bridges and saves it in the CI cache to make it available to other jobs,
+    # in particular the one building the AAR for binary distribution.
```

@mokagio I am not sure if I understand what you mean here, could you take a look at it? There are also a couple spelling mistakes.

---

@oguzkocer, sure!

This is the only spelling mistake I see.

```suggestion
    # This job builds the JS bundle that the Android react-native-bridge
    # bridges and saves it in the CI cache to make it available to other jobs,
    # in particular the one building the AAR for binary distribution.
```

What are the other(s)?

Aside from that, does this wording sound clearer?

```suggestion
    # This job builds the Android version of the JS bundle and saves it
    # in the CI cache to make it available to other jobs that might need
    # it, namely the job building the AAR for the Android version of the
    # react-native-bridge package.
```

---

Oh yes, this new version looks clearer to me.